src/handlers/render-questions.js

```diff
@@ -1,3 +1,25 @@
-export function renderQuestions(uiElement, questionsElement) {
-    uiElement.appendChild(questionsElement);
-}
+import { quizData } from "../data.js";
+
+export function renderQuestions() {
+    
+    const list = document.createElement('ul');
+    document.body.appendChild(list);
```

Comment:
Instead of creating the elements here in the handler, call `createQuestionsList` from `views/questions-list.js` and add the returned `<ul>` to the body here, in the handler.